### DISCUSSION TITLE
Phi-1.5 support?

### DISCUSSION THREAD

**Discussion by spikespiegel** [ggml-org/llama.cpp#3139] [Q&A]
It seems microsoft open-source phi-1.5 model (1.3B parameter model -> textbook is all you need) and put it in huggingface. (https://huggingface.co/microsoft/phi-1_5/tree/main), claiming comparable performance to 7B-13B models. Any chance to support this model?

**Comment by rayhern** [comment]
man oh man, im praying to the the llm gods now...

**Comment by chiefMarlin** [comment]
Any updates on this ?

**Comment by dmvaldman** [comment]
We can now skip straight to phi-2!

**Reply by kgoodridge** [reply]
Except Microsoft hasn't released the weights for phi-2, so that one would be extra challenging.

**Reply by kgoodridge** [reply]
Guess I'm wrong(ish)

https://twitter.com/SebastienBubeck/status/1734801722831413312?s=20

**Reply by liuzl** [reply]
https://huggingface.co/microsoft/phi-2

**Reply by aaryadav** [reply]
Quantized weights can be found here: 
https://huggingface.co/radames/phi-2-quantized

Can be run using Candle (rust ML framework)
https://github.com/huggingface/candle/tree/main/candle-examples/examples/phi

Would be great if it was supported here as well.

**Comment by KnutJaegersberg** [comment]
phi-2 would be a great addition. properly the best model of this size.

**Comment by niutech** [comment]
The progress can be tracked in the issues #3146 and #4437.
